e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -724,6 +725,77 @@ def test_set_workspace_namespace_bindings(
     output = detproc.check_output(sess, set_binding_cmd + ["--namespace", namespace])
     assert bound_to_namespace in output
 
+    # MultiRM: Valid cluster name, no namespace name, auto-create namespace & resource_quota.
```

Comment:
Can we do a couple of error cases?
Maybe setting a resource quota without specifying a namespace during workspace creation and verifying that it fails

Reply:
Sure thing! I'll also toss in an error case for negative values & text!

Reply:
Thanks!!

Reply:
Oh I do already have a case for that exact thing (not then invalid values):
```["det", "w", "create", w_name, "--resource-quota", "1"],```

I'll add one for defining a namespace & trying to set the resource quota!

Reply:
At some point it'll be worth while to come back and refactor these tests to use more structured testing design, but that can be resolved at a later date, I'll create a ticket for this